src/index.html

```diff
@@ -1,14 +1,70 @@
 <!doctype html>
-<html lang="en">
+<html lang = "en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset = "UTF-8">
+    <meta name = "viewport"
+          content = "width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv = "X-UA-Compatible" content = "ie=edge">
+    <link rel = "preconnect" href = "https://fonts.googleapis.com">
+    <link rel = "preconnect"
+      href = "https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href = "https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel = "stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel = "stylesheet" href = "./style.css">
```

Comment:
Avoid using unnecessary spaces around the '=' in HTML attributes. It will be same for all the cases